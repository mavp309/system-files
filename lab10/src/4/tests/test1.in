1234586 009100001 000000001
1234586 009100001 000000001
