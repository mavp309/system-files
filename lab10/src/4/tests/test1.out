2469172018200002000000002
